add version querying (#657)
PostgREST/postgrest-docs: docs/references/admin.rst

Server Version
--------------

> When debugging a problem it's important to verify the running PostgREST version. There are 3 ways you can check the version:

Reviewer: ```suggestion
When debugging a problem it's important to verify the running PostgREST version. There are three ways to do this:
```

> - Look for the :code:`Server` HTTP response header, which contains the version number. This header will be sent on all the requests.

Reviewer: ```suggestion
- Look for the :code:`Server` HTTP response header that PostgREST returns on every request.
```

I think that "which contains the version number" is implied in the previous paragraph.

Author: ```suggestion
- Look for the :code:`Server` HTTP response header that is returned on every request.
```

.. code::

  HEAD /users HTTP/1.1

  Server: postgrest/11.0.1

- Query ``application_name`` on `pg_stat_activity <https://www.postgresql.org/docs/current/monitoring-stats.html#MONITORING-PG-STAT-ACTIVITY-VIEW>`_.

.. code-block:: psql

>   select distinct application_name from pg_stat_activity where application_name ilike '%postgrest%';

Reviewer: ```suggestion
  select distinct application_name
  from pg_stat_activity
  where application_name ilike '%postgrest%';
```
